lib/rubocop/cop/commissioner.rb

```diff
@@ -11,10 +11,6 @@ class Commissioner
 
       attr_reader :errors
 
-      def self.callback_methods
```

Comment:
Hard to tell if other libraries rely on this, but I did a quick search in `rubocop-rspec`, which is probably our biggest API consumer, and it doesn't use it. I'd say it's safe to remove, and we'll see if someone opens an issue. 🙂